app/controllers/hubstats/deploys_controller.rb

```diff
@@ -81,6 +80,8 @@ def create
         @deploy.pull_requests = Hubstats::PullRequest.where(repo_id: @deploy.repo_id)
                                                      .where(number: @pull_request_id_array)
 
+        @deploy.user_id = @deploy.pull_requests.first.merged_by
```

Comment:
Not yet. I thought maybe we could get it to commissioner staging and see how the outcome was working. Although staging deploys don't include pull requests...